frontend/src/Components/CreateCommentComponent.tsx

```diff
@@ -79,6 +96,7 @@ export default function CreateCommentComponent(props: CreateCommentProps) {
     const [isPosting, setPosting] = useState(false);
     const [previewing, setPreviewing] = useState<string | null>(null);
     const [mediaUploaderOpen, setMediaUploaderOpen] = useState(false);
+    const hotKeysRef = useHotkeys<HTMLTextAreaElement>(allowedKeys.join(','), (_, e) => handleHotKey(e), {enableOnFormTags: true});
```

Comment:
Why global catch VS recommended pattern from documentation:
```
useHotkeys<HTMLTextAreaElement>('ctrl+enter,meta+enter', handleAnswer, {enableOnFormTags: true});
...
```
Afair it is more readable and extendable this way

Reply:
I think it's related to the focus:
https://github.com/JohannesKlauss/react-hotkeys-hook#focus-trap

Reply:
> I think it's related to the focus: https://github.com/JohannesKlauss/react-hotkeys-hook#focus-trap

Yes, it is.